web/js/components/map/ol-coordinates.js

```diff
@@ -4,7 +4,7 @@ import Coordinates from './coordinates';
 import util from '../../util/util';
 import { transform } from 'ol/proj';
 
-class OlCoordinates extends React.Component {
+export class OlCoordinates extends React.Component {
```

Comment:
Exporting as default and as class. Not a problem, just redundant. 
Can remove `export default OlCoordinates;`... (will probably need to change the import in the test as well)